### PR TITLE
user account service test implemented

### PR DESCRIPTION
**Pull request by sakurano92**
User account service test for few test that left out blank
has been implemented

### CODE REVIEW

File: app/wama.core.unit.test/Services/UserAccountServiceTests.cs [hunk 1]
```diff
@@ -117,22 +122,40 @@ public UserAccountServiceTests()
             await Task.Delay(1);
         }
 
-        [Fact]
-        public async Task SuspendUserAccountAsyncTest()
+        [Theory(DisplayName = "Approve UserAccounts")]
+        [InlineData("8521473", 1)]
```
Comment (dubeme) [inline app/wama.core.unit.test/Services/UserAccountServiceTests.cs]: Change the numbers to an actual boolean value (**_true_** or **_false_**)

**Review by dubeme — CHANGES_REQUESTED** [review]
Not bad, the main issue has to do with you using numbers for boolean. For readability, and reducing level of confusion, switch to using an actual boolean value (true or false).

File: app/wama.core.unit.test/Services/UserAccountServiceTests.cs (hunk 1 above)
Comment (sakurano92) [reply app/wama.core.unit.test/Services/UserAccountServiceTests.cs]: oh yea forgot about it

**Comment by iamcreasy** [comment]
expectedCount in GetPendingUserAccountsAsyncTest test is being testing against zero. Is this correct?

File: app/wama.core.unit.test/Services/UserAccountServiceTests.cs [hunk 2]
```diff
@@ -99,10 +99,15 @@ public UserAccountServiceTests()
...
         }
 
-        [Fact]
-        public async Task GetPendingUserAccountsAsyncTest()
+        [Theory(DisplayName = "Get pending UserAccounts")]
+        [InlineData(UserAccountType.Administrator, 0)]
+        [InlineData(UserAccountType.Employee, 0)]
+        [InlineData(UserAccountType.Manager, 0)]
+        [InlineData(UserAccountType.Patron, 0)]
+        public async Task GetPendingUserAccountsAsyncTest(UserAccountType accountType, int expectedCount)
```
Comment (quazi-irfan) [inline app/wama.core.unit.test/Services/UserAccountServiceTests.cs:107]: expectedCount in GetPendingUserAccountsAsyncTest test is being testing against zero. Is this correct?

**Review by dubeme — APPROVED** [review]

**Pull request merged**